plugins/repoquery.py

```diff
@@ -97,12 +100,16 @@ def parse_arguments(args):
                         help=_('show only results there provides REQ'))
     parser.add_argument('--whatrequires', metavar='REQ',
                         help=_('show only results there requires REQ'))
+    parser.add_argument("--alldeps", action="store_true",
+                        help="shows results that requires package provides and files")
     parser.add_argument('--querytags', action='store_true',
                         help=_('show available tags to use with '
                                '--queryformat'))
     parser.add_argument('--resolve', action='store_true',
                         help=_('resolve capabilities to originating package(s)')
-                       )
+                        )
+    parser.add_argument("--tree", action="store_true", 
+                        help="For the given packages print a tree of the packages.")
```

Comment:
Well, it is, but it could take argument from the ouform. In that case the outform couldn't be mutually exclusive group. (The argparse will make sure that only one of the arguments in the mutually exclusive group is present on the command line) https://docs.python.org/dev/library/argparse.html